app/src/interfaces/_system/system-filter/nodes.vue

```diff
@@ -282,6 +301,150 @@ function isExistingField(node: Record<string, any>): boolean {
 	const field = fieldsStore.getField(props.collection, fieldKey);
 	return !!field;
 }
+
+function getRelatedCollectionForField(fieldPath: string): string | null {
```

Comment:
This doesn't have any tests nor any description on what this does. The name of the function is also misleading.